packages/p2p/src/components/my-ads/toggle-ads.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes            from 'prop-types';
+import { localize }         from 'deriv-translations';
+import { ToggleSwitch }     from 'deriv-components';
+import classNames           from 'classnames';
+import './my-ads.scss';
+
+const ToggleMessage = props => {
+    return (
+        <p className={props.className}>
+            {props.is_enabled ? localize('Your ads are running') : localize('Your ads are paused')}
+        </p>
+    );
+};
+
+ToggleMessage.propTypes = {
+    className : PropTypes.string,
+    is_enabled: PropTypes.bool.isRequired,
+};
+
+class ToggleAds extends Component {
+
+    state = {
+        is_enabled: this.props.is_enabled,
+    }
+
+    setEnabled (is_enabled) {
+        this.setState({ is_enabled });
+    }
+
+    render () {
+        return (
+            <div className={classNames(
+                'toggle-ads',
+                {
+                    'toggle-ads--on' : this.state.is_enabled,
+                    'toggle-ads--off': !this.state.is_enabled,
+                }
```

Comment:
why don't we do `className={classNames('toggle-ads', this.state.is_enabled ? 'toggle-ads--on' : toggle-ads--off')}`
IMO that's more clear that only one of these states will be added